base-console-app-cert/Program.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Graph;
+using Microsoft.Extensions.Configuration;
+using System.Security.Cryptography.X509Certificates;
+
+namespace ConsoleGraphTest
+{
+    class Program
+    {
+        private static GraphServiceClient _graphServiceClient;
+        private static HttpClient _httpClient;
+
+        static void Main(string[] args)
+        {
+            // Load appsettings.json
+            var config = LoadAppSettings();
+            if (null == config)
+            {
+                Console.WriteLine("Missing or invalid appsettings.json file. Please see README.md for configuration instructions.");
+                return;
+            }
+
+            //Query using Graph SDK (preferred when possible)
+            GraphServiceClient graphClient = GetAuthenticatedGraphClient(config);
+            List<QueryOption> options = new List<QueryOption>
+            {
+                new QueryOption("$top", "1")
+            };
+
+            var graphResult = graphClient.Users.Request(options).GetAsync().Result;
+            Console.WriteLine("Graph SDK Result");
+            Console.WriteLine(graphResult[0].DisplayName);
+
+            //Direct query using HTTPClient (for beta endpoint calls or not available in Graph SDK)
```

Comment:
Why show both approaches? is it to show that customers can use the Graph SDK or REST APIs?

Reply:
Some of the later daily samples call endpoints that are not available through the Graph SDK (at least at the time of writing).  Additionally we wanted to show readers both options for making authenticated Graph calls.

Reply:
thanks for the update. This makes sense.